notebooks/rnn/lyft-rnn-lstm.ipynb

```diff
@@ -1 +1,1189 @@
\ No newline at end of file
+{
```

Comment:
@dcher95 are there any actual changes to this notebook. If so could you push the script version for easy review.

Reply:
No changes because I assumed the other notebook was the production script due to errors. Is this script for production?

Reply:
Yep, I think the other one is the prod script now 👍 . Thanks for confirming.

Reply:
Yep, I think the other one is the prod script now 👍 . Thanks for confirming.